src/ListItem/ListItem.react.js

```diff
@@ -268,9 +269,8 @@ class ListItem extends PureComponent {
                 secondaryText = centerElement.secondaryText;
                 tertiaryText = centerElement.tertiaryText;
             }
-            const numberOfLines = !tertiaryText ?
-                getNumberOfSecondaryTextLines(this.state.numberOfLines) : 1;
-
+            const secondLineNumber = typeof tertiaryText === 'undefined' ? numberOfLines : 1;
```

Comment:
what about `null`? or empty string? or false? think `!tertiaryText` it's better in this case ...